### ISSUE TITLE
Feature: Receiving emails and create fiiles / records in table

### ISSUE THREAD

**Issue by AleksandrSokolov** [saltcorn/saltcorn#322]
This is very intesting to create solutions like helpdesk or crm

**Comment by zinkh** [comment]
Can you explain how does this work? I guess the email must have a particular formatting so it can be understood by saltcorn and parsed into different rows of table.

**Comment by AleksandrSokolov** [comment]
@zinkh Yes, the idea is to have parser on saltcorn side and rules for email processing. May there are any simular open source tools / libraries. We need to make research.

Later I'll explain more details

**Comment by AleksandrSokolov** [comment]
@zinkh 

1. We need the service that will receive emails.
I,m not sure that is optimal but for example this service can work as email client and read emails from some email service using smtp.
So we nad to have second connection configuration for this smtp conneciton (We already have one for sending mails).
For first time it can be same smtp connection.

2. We need the rules based configurator for processing emails (simular to embeded into email clients).

For example, 
if the email title contains "Issue" than service will create issue and copy some email info to issues.
Title -> issue.title
email from -> issue.author.
email.body -> issue.description
email.senddate -> issue.senddate
email attachent list -> to additional table issue_attachments linked to issue.
